File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.cpp
class Solution {
public:
    int minBitFlips(int a, int b) {
        int cnt =0;
        while(a>0 or b>0 ){
            if((a&1)!=(b&1))
                ++cnt;
            a=a>>1;
            b=b>>1;
            // cout<<a<<" "<<b<<endl;
            // break;
        }
        return cnt;
    }
};

File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.py
class Solution:
    def minBitFlips(self, a: int, b: int) -> int:
        cnt = 0
        while(a>0 or b>0):
            if( (a&1)!=(b&1)):
                cnt+=1
            # print(a,b)
            a = a>>1
            b = b>>1
        return cnt